pysat/instruments/pysat_testing2d.py

```diff
@@ -66,19 +85,7 @@ def load(fnames, tag=None, sat_id=None, malformed_index=False):
     """
 
     # create an artifical satellite data set
-    parts = os.path.split(fnames[0])[-1].split('-')
-    yr = int(parts[0])
-    month = int(parts[1])
-    day = int(parts[2][0:2])
-    date = pysat.datetime(yr, month, day)
-    # scalar divisor below used to reduce the number of time samples
-    # covered by the simulation per day. The higher the number the lower
-    # the number of samples (86400/scalar)
-    scalar = 100
-    num = 86400/scalar
-    # basic time signal in UTS
-    uts = np.arange(num) * scalar
-    num_array = np.arange(num) * scalar
+    uts, index, date = mm_test.generate_times(fnames, sat_id, freq='100S')
```

Comment:
Why are we using 100 S here?

Reply:
I believe I reduced the sample rate here to keep simulation time down. I don't recall that it is a requirement.

Reply:
It causes Travis CI to stall (10 min with no output) if a full day at 1 s is used.